src/sub-blocks/BasicCard/BasicCard.scss

```diff
@@ -4,15 +4,42 @@
 $block: '.#{$ns}basic-card';
 
 #{$block} {
+    $iconSizeL: 32px;
```

Comment:
why do we have L and S sizes, but don't have m?

Reply:
I've changed it